Leaflet.VectorTiles.js

```diff
@@ -105,7 +296,7 @@ L.VectorTiles = L.GridLayer.extend({
     //     valid: <boolean>
     //   }
     // }
-    this._vectorTiles = {};
+    this._vectorTiles = window.vectorTiles = {};
```

Comment:
`window.vectorTiles` seems to be a debug-only object and not used anywhere else.